speedtest-tracker/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  speedtest-tracker:
+    image: ${SPEEDTEST_TRACKER_IMAGE}
+    restart: unless-stopped
+    #ports:
+    #    - 8080:80
+    #    - 8443:443
+    environment:
+      - PUID=${SPEEDTEST_TRACKER_UID}
+      - PGID=${SPEEDTEST_TRACKER_GID}
+      - APP_KEY=${SPEEDTEST_TRACKER_APP_KEY}
+      - DB_CONNECTION=sqlite
+      - ADMIN_NAME=${SPEEDTEST_TRACKER_ADMIN_NAME}
+      - ADMIN_EMAIL=${SPEEDTEST_TRACKER_ADMIN_EMAIL}
+      - ADMIN_PASSWORD=${SPEEDTEST_TRACKER_ADMIN_PASSWORD}
+      - DISPLAY_TIMEZONE=${SPEEDTEST_TRACKER_DISPLAY_TIMEZONE}
+      - APP_URL=${SPEEDTEST_TRACKER_TRAEFIK_HOST}
+      - ASSET_URL=${SPEEDTEST_TRACKER_TRAEFIK_HOST}
+    volumes:
+      - data:/config
+      - keys:/config/keys
```

Comment:
I'm guessing the keys mount is to hold the TLS key, which is not needed.

Reply:
I don't see an env var or config to disable nginx's TLS. If you don't create the keys volume, it still creates its own keys but I guess just stores them ephemerally in the container. I'm sure it's possible to build the image and have Dockerfile go in and change an nginx config, but at this point it's not worth it to me - too much effort for something i may never use or need.